cargo-shuttle/src/client.rs

```diff
@@ -87,11 +89,12 @@ pub(crate) async fn shuttle_version(mut api_url: ApiUrl) -> Result<String> {
     if response_status == StatusCode::OK {
         Ok(res.text().await?)
     } else {
-        Err(anyhow!(
-            "status: {}, body: {}",
-            response_status,
-            res.text().await?
-        ))
+        error!(
+            text = res.text().await?,
+            status = %response_status,
+            "failed to get shuttle version from server"
+        );
+        Err(anyhow!("failed to get shuttle version from server",))
```

Comment:
Same question as a comment below?

Reply:
Not quite, there is no `Result` here to call `.context()` on - ie nothing to "backtracify" so nothing to make nicer